doc/developers/advanced_installation.rst

```diff
@@ -353,7 +353,8 @@ Linux compilers from conda-forge
 Alternatively, install a recent version of the GNU C Compiler toolchain (GCC)
 in the user folder using conda::
 
-    conda create -n sklearn-dev numpy scipy joblib cython conda-forge::compilers
+    conda create -n sklearn-dev -c conda-forge python numpy scipy cython \
+        joblib threadpoolctl pytest compilers
```

Comment:
```suggestion
        joblib threadpoolctl compilers
```

Reply:
I think it's better to keep pytest as it's a convenience line people will copy and paste to get their dev environment setup for sprints (and we had it for macOS).

Reply:
ok